app/controllers/checklist_tests_controller.rb

```diff
@@ -6,9 +6,7 @@ class ChecklistTestsController < ProductTestsController
   before_action :set_measures, only: [:show]
 
   def create
-    # bundle ATTRIBUTE IS NOT CHOSEN CORRECTLY. MUST FIX LATER ~ JAEBIRD
```

Comment:
Oh I forgot about this file. Actually I don't think this class is even used anymore.  But the bundle is now tied to the product not the product test, so there's no need to specify the bundle id here.